Makefile

```diff
@@ -18,12 +18,13 @@ ifeq ($(origin IMG),undefined)
 ifeq ($(shell git branch --show-current),master)
 IMG=quay.io/dynatrace/dynatrace-operator:snapshot
 else
-IMG=quay.io/dynatrace/dynatrace-operator:snapshot-$(shell git branch --show-current | sed "s\#[^a-zA-Z0-9_-]\#-\#g")
+IMG=quay.io/dynatrace/dynatrace-operator:snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
 endif
 endif
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=true, preserveUnknownFields=false, crdVersions=v1"
+CRD_OPTIONS_OCP311 ?= "crd:trivialVersions=true, preserveUnknownFields=false, crdVersions=v1beta1"
```

Comment:
That blanks are adventurous. This string must be treated as one argument to controller-gen.

Reply:
Removed the blanks